jujugui/static/gui/src/app/app.js

```diff
@@ -887,11 +887,14 @@ YUI.add('juju-gui', function(Y) {
     _renderCharmbrowser: function(metadata) {
       var state = this.state;
       var utils = views.utils;
-      var cs = this.get('charmstore');
+      var charmstore = this.get('charmstore');
       React.render(
         <components.Charmbrowser
-          charmstoreSearch={cs.search.bind(cs)}
-          getEntity={cs.getEntity.bind(cs)}
+          charmstoreSearch={charmstore.search.bind(charmstore)}
+          importBundleYAML={this.bundleImporter.importBundleYAML.bind(
```

Comment:
Here we're just passing the reference to the `importBundleYAML` method. If need be we can do some error handling where we call the method here: https://github.com/juju/juju-gui/pull/910/files#diff-7cf7695007c40f36e1ca92951612a296R91